fe/fe-core/src/main/java/org/apache/doris/catalog/Column.java

```diff
@@ -159,6 +159,7 @@ public Column(Column column) {
     public void createChildrenColumn(Type type, Column column) {
         if (type.isArrayType()) {
             Column c = new Column(COLUMN_ARRAY_CHILDREN, ((ArrayType) type).getItemType());
+            c.setIsAllowNull(true);
```

Comment:
why always allow null

Reply:
I'm not sure whether there is a way to declare the item type in array type nullable.

Reply:
add a todo comment to his

Reply:
Done.